src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
```

Comment:
Critical: enum constants should be capital https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names